### PR TITLE
Feature/eslint

### PR DESCRIPTION
**Pull request by jason-fox**
Update codebase to use ES6
-  Remove JSHint and jshint overrides
-  Add esLint using standard tamia presets
-  Replace var with let/const
-  Fix or disable eslint errors

### CODE REVIEW

File: lib/sigfoxServer.js [hunk 1]
```diff
@@ -61,8 +61,8 @@ function traceRequest(req, res, next) {
 }
 
 function start(newConfig, callback) {
-    var baseRoot = '/',
-        agentName = 'default';
+    let baseRoot = '/';
+    //    let agentName = 'default';
```
Comment (fgalan) [inline lib/sigfoxServer.js:65]: Overcommented?

File: lib/sigfoxServer.js [hunk 2]
```diff
@@ -83,9 +83,9 @@ function start(newConfig, callback) {
         baseRoot = config.getConfig().iota.server.baseRoot;
     }
 
-    if (config.getConfig().iota.server.name) {
-        agentName = config.getConfig().iota.server.name;
-    }
+    //if (config.getConfig().iota.server.name) {
+    //    agentName = config.getConfig().iota.server.name;
+    //}
```
Comment (fgalan) [inline lib/sigfoxServer.js:88]: Commented code should be commented out?

Comment (jason-fox) [reply lib/sigfoxServer.js]: Fixed cd658dd

File: lib/sigfoxServer.js (hunk 1 above)
Comment (jason-fox) [reply lib/sigfoxServer.js]: Fixed cd658dd

**Pull request merged**